share/picongpu/tests/KHI_growthRate/include/picongpu/param/png.param

```diff
@@ -0,0 +1,100 @@
+/* Copyright 2013-2022 Axel Huebl, Heiko Burau, Rene Widera, Richard Pausch
```

Comment:
not needed for the CI - there is no png output

Reply:
I would keep it in if we need to debug the code